public/js/script.js

```diff
@@ -63,6 +90,10 @@ function loadStops() {
   });
 }
 
+/**
+ * @function loadLines
+ * @description when invoked loadLines() loads the long and lang coordinates from the http get request followed by mapping of the lineColors object which then draws polyline overlays on the map
```

Comment:
Same thing about not making requests to the server.

Reply:
@davis-trem for this one I will do my best to come up with a description and push up the code and wait for your review and suggestions for changes.